README.md

```diff
@@ -0,0 +1,121 @@
+
+[clonkspot.org](https://clonkspot.org) hosts the community forum, content developer documentation and masterserver.
+
+## Compiling - Quick Start
+**Disclaimer**: This readme is fairly new and may contain mistakes. In case of problems [reach out](#Contact). Readme improvement PRs are welcome.
+
+Essential dependencies:
+- CMake
+- Fairly modern C++ compiler
+	- Windows: **Latest MSVC** (mingw-w64 g++ ≥ 13.1 *should* work with cmake option `-DUSE_WINDOWS_RUNTIME=OFF`, but is *not tested regularly*)
+	- Linux: **g++ ≥ 13.1** or **clang++ ≥ 18.1**
+	- macOS: **open source clang++ ≥ 18.1** - e.g. `brew install llvm@18 ninja`
+- Make or Ninja
+
+Extract the [latest pre-built dependencies](https://github.com/legacyclonk/deps/releases/latest) for your platform into a folder called `deps`.
+Make sure that besides the `CMakeLists.txt` of LegacyClonk there are folders `deps/include`, `deps/lib`, etc. as extracted from the binary package.
+
+
+### Configuring and compiling the engine
+Open a Terminal and navigate to the repository root, such that `CMakeLists.txt` is in the current directory.
+
+#### Windows
+Use the MSVC developer PowerShell.
+
+Configure with CMake
+```PowerShell
+cmake . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
+```
+
+Open the resulting Visual Studio solution or simply compile with CMake as follows
+```PowerShell
+cmake --build build
+```
+
+#### Linux
+Configure with CMake
+```bash
+cmake . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWITH_DEVELOPER_MODE=ON
+```
+
+Compile with CMake
+```bash
+cmake --build build
+```
+
+#### macOS
+Configure with CMake (FIXME: Rename toolchain file, the version number doesn’t matter)
+```bash
+cmake . -B build -DCMAKE_TOOLCHAIN_FILE=$PWD/autobuild/clang16_mac.cmake
+```
+
+Compile with CMake
+```bash
+cmake --build build
+```
+
+Create the `.app`-bundle
+```bash
+DESTDIR=build/destdir cmake --build build --target install
+```
+
+The `clonk.app` should be generated at `build/destdir/usr/local/clonk.app`.
+
+
+### Setup game folder
+The engine needs various files to run placed next to the engine binary.
+The cleanest way is to create and setup a game folder somewhere outside of the source code repository.
+
+If you want to use an existing game folder (e.g. obtained via the simple binary [Installation](#Installation)) use that folder instead and only do step 2.
+
+1. Create an empty folder at the desired location.
+The guide assumes the directory path to be `~/clonk`.
+1. Create a symbolic link to the binary in the folder
+	- Windows: `mklink /H ~/clonk/MyClonk.exe build/clonk.exe`
+	- Linux: `ln -s build/clonk ~/clonk/myclonk`
+	- macOS: `ln -s build/clonk.app ~/clonk/myclonk.app`
+1. Create symbolic links to [Graphics.c4g](planet/Graphics.c4g) and [System.c4g](planet/System.c4g). Without them, the engine won’t start.
+	- Windows
+	```PowerShell
+	mklink /J ~/clonk/Graphics.c4g planet/Graphics.c4g
+	mklink /J ~/clonk/System.c4g planet/System.c4g
```

Comment:
PowerShell actually doesn't know `mklink` - this should be
```suggestion
	New-Item -ItemType Junction -Path ~/clonk/Graphics.c4g -Target planet/Graphics.c4g
	New-Item -ItemType Junction -Path ~/clonk/System.c4g -Target planet/System.c4g
```

Reply:
Good point. Do you know how to link multiple items with wildcard? (Regarding content *.c4?)